tests/unit/plugins/modules/test_node_settings_should_format_well_the_queries.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
Is the fact that there are no shebang on purpose?

Reply:
Yes it is not meant to be executable, it should be run through pytest or ansible-test